services/cloud/cloud.go

```diff
@@ -248,3 +248,34 @@ type Cluster struct {
 	// Nodes
 	Nodes *[]Node `json:"nodes,omitempty"`
 }
+
+// ControlPlaneProperties the resource group properties.
+type ControlPlaneProperties struct {
+	// State - State
+	Statuses map[string]*string `json:"statuses"`
+	// Fqdn
+	Fqdn *string `json:"fqdn,omitempty"`
+	// Port
+	Port *int32 `json:"port,omitempty"`
+	// AuthorizerPort
```

Comment:
Can you please make these comments more descriptive?

Reply:
will do.